src/test/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/writetag/InputWriteTag.java

```diff
@@ -6,30 +6,34 @@
 package com.puppycrawl.tools.checkstyle.checks.javadoc.writetag;
 
 /**
+ * Config:
+ * tag = &#64;author
+ * tagFormat = \\S
+ *
  * Testing tag writing
  * @author Daniel Grenner
  * @incomplete This class needs more code...
  * @doubletag first text
  * @doubletag second text
  * @emptytag
  */
-class InputWriteTag
+class InputWriteTag // violation at 14
 {
-        /**
-         * @todo Add a constructor comment
-         */
-        public InputWriteTag()
-        {
-        }
+    /**
+     * @todo Add a constructor comment
+     */
+    public InputWriteTag() // ok
```

Comment:
you do not have to write trailing `// ok` comments, based on [these suggestions](https://github.com/checkstyle/checkstyle/wiki/Good-practice-on-how-to-create-Input-files-for-tests), only `violations`

Reply:
Sir, this is true, but what about files which have no violations. Also, we have been following this practice so I prefer to keep this. If you insist I would remove them, but for the time being I have kept them.

Reply:
Sir, please share your final views on this.

Reply:
We require one occurrence of ` * Config` and one occurrence of  `// ok` or `// violation` in all check input files:  https://github.com/checkstyle/checkstyle/blob/81df29c69f57579f247b76bafdf1393cc28a1997/config/checkstyle_input_checks.xml#L19